### ISSUE TITLE
The same seed and prompt produces a completely black image

### ISSUE THREAD

**Issue by edisonzf2020** [segmind/segmoe#3]
I use the following code to test

```
from segmoe import SegMoEPipeline

pipeline = SegMoEPipeline("segmind/SegMoE-sd-4x2-v0", device = "cuda")

prompt = "a beautiful Asian girl, front view, full-length portrait, One hand is half-raised, as if waving hello."
negative_prompt = "ugly, deformed, badhandv4, ng_deepnegative_v1_75t, nsfw, bad quality, worse quality"
img = pipeline(
    prompt=prompt,
    negative_prompt=negative_prompt,
    height=768,
    width=512,
    num_inference_steps=30,
    guidance_scale=7.5,
    seed=1575769330,
    Sampler='DPM++ SDE Karras',
    low_cpu_mem_usage=True,
).images[0]
img.save("image1.png")
```
An error will occur if you repeat the execution 2-3 times.
The error message is as follows:

`Potential NSFW content was detected in one or more images. A black image will be returned instead. Try again with a different prompt and/or seed.`

**Comment by Warlord-K** [comment]
Please try this, 
```python
from segmoe import SegMoEPipeline

pipeline = SegMoEPipeline("segmind/SegMoE-sd-4x2-v0", device = "cuda", safety_checker = None)

prompt = "a beautiful Asian girl, front view, full-length portrait, One hand is half-raised, as if waving hello."
negative_prompt = "ugly, deformed, badhandv4, ng_deepnegative_v1_75t, nsfw, bad quality, worse quality"
img = pipeline(
    prompt=prompt,
    negative_prompt=negative_prompt,
    height=768,
    width=512,
    num_inference_steps=30,
    guidance_scale=7.5,
    seed=1575769330,
    Sampler='DPM++ SDE Karras',
    low_cpu_mem_usage=True,
).images[0]
img.save("image1.png")
```

**Comment by Vargol** [comment]
HI, I
 tried this yesterday and double checked just now and it doesn't work.
The SegMoEPipeline doesn't pass the kwargs into the DiffusionPipeline

**Comment by Gothos** [comment]
Hello, could you please elaborate on the issue? Any stacktraces would help, as would further details of where the kwargs are missed.

**Comment by Warlord-K** [comment]
Fixed with PR #14, Thanks for raising the issue!
